source/agora/consensus/protocol/Nominator.d

```diff
@@ -168,7 +244,8 @@ extern(D):
 
     ***************************************************************************/
 
-    protected bool prepareNominatingSet (out ConsensusData data) @safe
+    protected bool prepareNominatingSet (out ConsensusData data,
```

Comment:
Needs `Params` docs. `cur_time` is used by derived classes, and will also be used here but by subsequent PRs.